nixos/tests/web-apps/mastodon/mastodon-std.nix

```diff
@@ -144,8 +142,12 @@ in
     server.wait_for_open_port(55000)
     server.wait_for_open_port(55001)
 
-    # Check Mastodon version from remote client
+    # Check that mastodon-media-auto-remove is scheduled
+    server.succeed("systemctl status mastodon-media-auto-remove.timer")
+
+    # Check work Mastodon from remote client
     client.succeed("curl --fail https://mastodon.local/api/v1/instance | jq -r '.version' | grep '${pkgs.mastodon.version}'")
+    client.succeed("curl --fail $(curl https://mastodon.local/api/v1/instance 2> /dev/null | jq -r .thumbnail) --output /dev/null")
```

Comment:
This is good, but it's orthogonal to this PR. Maybe we can merge this, and all other changes that are unrelated to the remote db first in a separate PR? It would be easier because those are changes that everyone can easily agree upon, so it would be merged quicker.

Reply:
Moved changes to another PR.